segments/path.bash

```diff
@@ -11,15 +12,20 @@ function _sbp_generate_path_segment {
   sep_color=$(print_color_escapes "$_sbp_path_color_sep" "$_sbp_path_color_bg")
   sep=" ${sep_color}${_sbp_char_path}${path_color} "
   wdir=${PWD/$HOME/\~}
-  IFS=/ read -r -a wdir_array <<<"$wdir"
-  if [[ ${#wdir_array[@]} -gt 1 ]]; then
-    for folder in "${wdir_array[@]}"; do
-      path_length=$(( path_length + ${#folder} + 3 ))
-    done
-    path_length=$(( path_length - 2 ))
-    path_value=" ${wdir//\//$sep} "
+  if [ $_sbp_path_single_seg -eq 0 ]; then
```

Comment:
I think double [[ and  quotes around the left hand side would be a good convention here. Also, in my head 0 = false and 1 = true, at least that seems to be the most used convention. So I think we should change that, what do you think?